crypto/dilithium/p_dilithium3_asn1.c

```diff
@@ -182,10 +182,11 @@ static int dilithium3_bits(const EVP_PKEY *pkey) {
 
 const EVP_PKEY_ASN1_METHOD dilithium3_asn1_meth = {
     EVP_PKEY_DILITHIUM3,
-    //1.3.6.1.4.1.2.267.7 = Dilithium SIG Round-3. These are temp values from
-    //http://oid-info.com/get/1.3.6.1.4.1.2.267.7 as we await NIST to release OIDs.
-    {0x2B, 0x06, 0x01, 0x04, 0x01, 0x02, 0x82, 0x0B, 0x07},
-    9,
+    // 1.3.6.1.4.1.2.267.7.6.5 = Dilithium SIG Round-3. These are temp values from
+    // https://github.com/IETF-Hackathon/pqc-certificates/blob/master/docs/oid_mapping.md
+    // as we await NIST to release OIDs.
+    {0x2B, 0x06, 0x01, 0x04, 0x01, 0x02, 0x82, 0x0B, 0x07, 0x06, 0x05},
+    11,
```

Comment:
How is this encoded? How can I verify this is the correct value? We have 4 copies of this value in the code and this one appears to be a different value than the other base 10 values.

Reply:
See `8.19 Encoding of an object identifier value` of [X.690 : Information technology - ASN.1 encoding rules: Specification of Basic Encoding Rules (BER), Canonical Encoding Rules (CER) and Distinguished Encoding Rules (DER)](https://www.itu.int/rec/T-REC-X.690-202102-I/en).

```
Two first numbers 1.3 encoded as: x.y -> (40*x)+y. Thus 40*1+3 = 43 -> 0x2B.

6 is encoded as 0x06
1 is encoded as 0x01
4 is encoded as 0x04
1 is endoded as 0x01
2 is encoded as 0x02
267 is 0x10B, so we use must 2 bytes to encode,

"Conceptually, these groups of bits are concatenated to form an unsigned binary number whose most significant bit is bit 7 of the first octet and whose least significant bit is bit 1 of the last octet."

so we want to encode 100001011 to 2 hex bytes, first split into 7 bits: 0000010 0001011
first cluster is not the last in the sequence, so we prepend with a 1: 10000010 -> 0x82 in hex
last cluser is the last in the sequence to we prepend with a 0:        00001011 -> 0x0B in hex

267 is encoded as 0x82 0x0B
7 is encoded as 0x07
6 is encoded as 0x06
5 is encoded as 0x05
```